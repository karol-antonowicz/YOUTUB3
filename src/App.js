import React from 'react';
import HeaderNav from './containers/HeaderNav/HeaderNav'
import { SideBar } from './containers/SideBar/SideBar';
import { Home } from './components/Home/Home';

const App = () => {
  return (
    <React.Fragment>
<HeaderNav/>
<Home/>
    </React.Fragment>
  );
}

export default App;
