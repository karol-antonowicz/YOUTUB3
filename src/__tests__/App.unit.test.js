import React from 'react';
import App from '../App';
import {shallow} from 'enzyme';

describe('App', ()=> {
    test('Renders', ()=> {
        const wrapper = shallow(<App/>);
        expect(wrapper).toMatchSnapshot();
    });
});