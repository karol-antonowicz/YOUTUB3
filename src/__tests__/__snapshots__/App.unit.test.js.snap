// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders 1`] = `
<AppLayout>
  <Switch>
    <Route
      component={[Function]}
      path="/watch"
    />
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</AppLayout>
`;
